mmv1/third_party/terraform/services/container/node_config.go.erb

```diff
@@ -413,26 +413,26 @@ func schemaNodeConfig() *schema.Schema {
 				"taint": {
 					Type:        schema.TypeList,
 					Optional:    true,
-					ForceNew:    true,
+					ForceNew:    false,
```

Comment:
These can just be removed rather than ForceNew: false

Reply:
requested changes pushed